### DISCUSSION TITLE
Reconnect to a server after being kicked (out of the server not to the hub)

### DISCUSSION THREAD

**Discussion by AmGecko** [ORelio/Minecraft-Console-Client#2370] [Q&A] [answered]
The server I go on goes down sometimes due to development so is there a way for my alt to periodically attempt to reconnect to a server after being kicked (from the server not to the hub)

**Comment by milutinke** [comment, marked as answer]
> The server I go on goes down sometimes due to development so is there a way for my alt to periodically attempt to reconnect to a server after being kicked (from the server not to the hub)

https://mccteam.github.io/guide/chat-bots.html#auto-relog
